pkg/security/tests/module_tester.go

```diff
@@ -412,6 +412,12 @@ func assertTriggeredRule(tb testing.TB, r *rules.Rule, id string) bool {
 	return assert.Equal(tb, id, r.ID, "wrong triggered rule")
 }
 
+//nolint:deadcode,unused
```

Comment:
i would have used a different message like `did not trigger rule %s` or something like that to differentiate from the other assertion that has the same message